test/models/related_topic_list_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+
+describe RelatedTopicList do
+  describe "#related_topics_for" do
+    it "returns related topics for a subsection" do
+      content_store = content_store_with_response(links: {related_topics: [{ title: 'From Content Store', web_url: 'http://example.org/whatever'}]})
+
+      topic_list = RelatedTopicList.new(content_store, nil)
```

Comment:
I think it would be good to check here that, if the content_store service returns something, the whitehall service isn't even called.